modules/features/cu_ldap/cu_ldap.module

```diff
@@ -122,14 +121,19 @@ function cu_ldap_user_login(&$edit, $account) {
 /**
  * Implements hook_cron().
  */
-
-function cu_ldap_cron(){
+function cu_ldap_cron() {
   // Only need to do this once a month.
-  if (REQUEST_TIME >= strtotime('+1 month', (int) variable_get('cu_ldap_ping_time', strtotime('-1 year')))) {
+  $last_cron_run = variable_get('cu_ldap_ping_time');
+  if (!isset($last_cron_run)) {
+    variable_set('cu_ldap_ping_time', 0);
+  }
+  if (REQUEST_TIME > strtotime('+1 month', (int) variable_get('cu_ldap_ping_time', strtotime('-1 year')))) {
```

Comment:
Variable_get requires a default https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/variable_get/7.x

Not including one in the initial $last_cron_run = variable_get and then adding an isset check is already unnecessary code, but Drupal is writing syslog errors every time variable_get is called without a default and then a different default is added in line 130 that can never be used because of the lines above it.